libs/account/src/state.ts

```diff
@@ -26,7 +26,7 @@ export type AccountState = {
 
 
  // IDX vars
- idx?: IDX;
+ idx: IDX|null;
```

Comment:
Try to use `null` for things that are there, but null, and `undefined` for things that are not found